.github/actions/action-setup_task-installPyProject/action.yml

```diff
@@ -7,17 +7,42 @@ inputs:
     description: Python version to setup
     required: true
   cache-id:
-    description: Reference id to define cache
+    description: |
+      Reference id to define cache. This id should be unique to the commit or PR
+      (e.g. the PR id or git hash)
     required: false
     default: ${{ github.event.pull_request.id || github.event.after }}
+  cache-key:
+    description: |
+      Unique key to identify cache. This id should be unique to the job step
+      (e.g. 'test', 'docs')
+    required: false
   manager:
     description: Dependency manager used (e.g. poetry)
     required: false
     default: "poetry"
   install-library:
```

Comment:
Are all the parameters here and below suppose to act as boolean? All these parameters by default are strings with reusable actions - I'm not sure if this changes the conditionals at all below.

For example, if someone was to input "abc" into `install-deps-with`, that would also trigger that condition. We may want to throw a warning message to console if an "unexpected" input (will have to define) is provided (I probably should have added something in previously too).

Reply:
I guess my question is more, what is the expected input? Is the intention to assume any string input to trigger the associated condition?

Reply:
The only parameter of which that's true is `install-deps-only`, which will cause `install-deps-with` to be ignored if provided

Reply:
Fair enough - I just took another look at the code base and I think this is fine as is then. Didn't fully appreciate how these parameters were being used.

Reply:
What newlines? Sorry I think I missed a review

Reply:
Just adding a newline between steps - minor thing just to make the workflow slightly easier to read

Reply:
A gotcha, sorry, failed to fully read your initial comment